CONTRIBUTING.md

```diff
@@ -1,50 +1,161 @@
-# Contributing to ODE Test Problems
+# Contributing to `ODE Test Problems`
 
-This guide provides instructions for submitting and formatting new code in OTP.
+This guide provides instructions for submitting and formatting new code in `OTP`. 
 
 ## Submitting Changes
 
-Changes to OTP should be proposed as a pull request and undergo a review process before being merged into master. New
-code must be free of warnings and errors and adhere to the style guidelines.
+Changes to `OTP` should be proposed as a pull request and undergo a review process before being merged. New code must be free of warnings and errors and adhere to the style guidelines (see also [The style guidelines](## Style guidelines)).
 
 ## Creating a Problem
 
-Each problem defines a package under `+otp` that contains all files used by the problem. When creating a new problem, we
-recommend duplicating an existing problem package, then renaming and editing the contents as needed.
+Each problem defines a package under `+otp` that contains all files used by the problem. To add a new test problem follow these steps:
+
+1. Check out the latest version of `OTP`:
+
+```bash
+git clone https://github.com/ComputationalScienceLaboratory/ODE-Test-Problems.git
+cd ODE-Test-Problems/
+```
+
+2. Create a new folder in the `src/+opt/` directory. Follow the same naming conventions as the existing problems. Start the name with `+` to maintain the structure of the Matlab/Octave package. Also create a subfolder named `+presets` in the new problem folder:
+
+```bash
+mkdir src/+opt/+newtest
+mkdir src/+opt/+newtest/+presets
+cd src/+opt/+newtest/
+```
+
+3. The minimal set of files needed inside the problem folder to set up a new test problem are:
+  *  The right-hand-side function named as `f.m` 
```

Comment:
I feel like explicitlly give commands to create those files, so either do vim/nano, or touch or maybe the way to open with matlab?